modules/release-notes/con-release-notes-overview.adoc

```diff
@@ -19,3 +19,22 @@ Benefits of {product} include:
 .Additional resources
 
 * link:{upgrading-book-url}[{upgrading-book-title}].
+
+== Supported Platforms
+
+{product} runs on OpenShift Container Platform 4.14-4.16 on the following CPU architecture:
```

Comment:
@hmanwani-rh, I've written the CPU architecture as an unordered list item even though it's just one item based on [this](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#length-of-lists) from the IBM Style Guide:
"Use at least two list items in an ordered list. You can have **one item in an unordered list to match the format of other unordered lists in the same area**."

Reply:
Use the ocp-version-min and ocp-version attributes

Reply:
@themr0c , I updated the ocp-version-min and ocp-version attributes and used them. Thank you! 
I also updated "OpenShift Container Platform" with ocp-short.